### PR TITLE
🌐 Add French translation for `docs/fr/docs/tutorial/path-params-numeric-validations.md`

### PR DESCRIPTION
**Pull request by p3bri4n**
Bonjour 🇫🇷 🤗

Here is the PR to translate the Path Parameters and Numeric Validations tutorial.
See the french translation tracking issue https://github.com/tiangolo/fastapi/issues/1972.

Thanks for the reviews 🙏

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3569e8a30ceba7da1dd9e988f6d4bd342ea36605 at: https://76e8697c.fastapitiangolo.pages.dev

File: docs/fr/docs/tutorial/path-params-numeric-validations.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+!!! info
+    FastAPI a ajouté le support pour `Annotated` (et a commencé à le recommander) dans la version 0.95.0.
+
+    Si vous avez une version plus ancienne, vous obtiendrez des erreurs en essayant d'utiliser `Annotated`.
+
+    Assurez-vous de [Mettre à jour la version de FastAPI](../deployment/versions.md#upgrading-the-fastapi-versions){.internal-link target=_blank} à la version 0.95.1 à minima avant d'utiliser `Annotated`.
+
+## Déclarer des métadonnées
+
+Vous pouvez déclarer tous les mêmes paramètres que pour `Query`.
```
Comment (mariot) [inline docs/fr/docs/tutorial/path-params-numeric-validations.md:54]:
```suggestion
Vous pouvez déclarer les mêmes paramètres que pour `Query`.
```

File: docs/fr/docs/tutorial/path-params-numeric-validations.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+Par exemple, pour déclarer une valeur de métadonnée `title` pour le paramètre de chemin `item_id`, vous pouvez taper :
```
Comment (mariot) [inline docs/fr/docs/tutorial/path-params-numeric-validations.md:56]:
```suggestion
Par exemple, pour déclarer une valeur de métadonnée `title` pour le paramètre de chemin `item_id`, vous pouvez écrire :
```

File: docs/fr/docs/tutorial/path-params-numeric-validations.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+
+    !!! tip
+        Préférez utiliser la version `Annotated` si possible.
+
+    ```Python hl_lines="10"
+    {!> ../../../docs_src/path_params_numeric_validations/tutorial001.py!}
+    ```
+
+!!! note
+    Un paramètre de chemin est toujours requis car il doit faire partie du chemin. Même si vous l'avez déclaré avec `None` ou défini une valeur par défaut, cela n'affecterait rien, il serait toujours requis.
```
Comment (mariot) [inline docs/fr/docs/tutorial/path-params-numeric-validations.md:95]:
```suggestion
    Un paramètre de chemin est toujours requis car il doit faire partie du chemin. Même si vous l'avez déclaré avec `None` ou défini une valeur par défaut, cela ne changerait rien, il serait toujours requis.
```

**Review by mariot — APPROVED** [review]
C'est super! Merci beaucoup.
J'ai juste mis quelques suggestions :)

**Comment by pe-brian** [comment]
Très bonnes suggestions,  merci bien !

**Comment by alejsdev** [comment]
Nice! Thanks for your contribution @pe-brian :v: 
And thanks @mariot for your help :tada:

**Pull request merged**